MonoGame.Framework/Matrix.cs

```diff
@@ -545,14 +732,32 @@ public static void CreateFromQuaternion(ref Quaternion quaternion, out Matrix re
 		    result.M43 = 0f;
 		    result.M44 = 1f;
         }
-		
+
+        /// <summary>
+        /// Creates a new rotation <see cref="Matrix"/> from the specified yaw,pitch and roll values.
+        /// </summary>
+        /// <param name="yaw">The yaw rotation value.</param>
+        /// <param name="pitch">The pitch rotation value.</param>
+        /// <param name="roll">The roll rotation value.</param>

```

Comment:
Yaw, pitch, and roll are all rotation angles... are they supposed to be in radians or degrees?